basictracer/tracer.py

```diff
@@ -116,23 +115,6 @@ def start_manual(
             tags=tags,
             start_time=start_time)
 
-    def start_span(
```

Comment:
are we dropping entirely the `start_span`? I mean with this change current instrumentation will have a different behavior, correct?